src/App.js

```diff
@@ -289,7 +323,7 @@ export default class App extends Lightning.Component {
         $enter() {
           Log.info('\n In App State ')
           this.tag('LivePlayback').$stop()
-          this.visible = false

```

Comment:
Is it when we enter premium/ metro apps state, the UI visibility should remain false , right

Reply:
yes